faceted_search/src/solr-library-extension.js

```diff
@@ -0,0 +1,106 @@
+// Purpose: This file is to export iSamples customized UI functions
```

Comment:
I think we should try and rewrite these as React components and use the custom components functionality of `Solr-faceted-search-react`: https://www.npmjs.com/package/solr-faceted-search-react#injecting-custom-components

Reply:
As a general guideline, we should try and work with React rather than falling down to the DOM level.  This might not always be possible or practical, but in this case I think it should be pretty simple to adapt the highlighting and linking logic into a React component.

Reply:
Reasonable. I will try to modify it to React components today.